ts/features/payments/checkout/saga/networking/handleWalletPaymentCreateTransaction.ts

```diff
@@ -32,14 +37,21 @@ export function* handleWalletPaymentCreateTransaction(
       paymentsCreateTransactionAction.failure,
       action,
       {
-        body: action.payload
+        body: action.payload.data
```

Comment:
Do we need this to fix a bug?

Reply:
Yes we need it since the payload has changed adding the `onError` callback that is used to reset a state in the View